RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -191,66 +186,66 @@ private void initializeViews(){
         mNewProjectButton.setOnClickListener(btnClick);
 
         hideProjectsIfEmpty(mNumProjects);
-        if(mNumProjects > 0){
+        if (mNumProjects > 0) {
             initializeRecentProject();
-            if(mNumProjects > 1) {
+            if (mNumProjects > 1) {
                 populateProjectList();
             }
         } else {
             mProjectList.setVisibility(View.GONE);
         }
     }
 
-    public void initializeRecentProject(){
+    public void initializeRecentProject() {
         Project project = null;
-        if(pref.getString(Settings.KEY_PREF_LANG, "").compareTo("") != 0) {
+        if (pref.getString(Settings.KEY_PREF_LANG, "").compareTo("") != 0) {
             project = Project.getProjectFromPreferences(this);
         } else {
             ProjectDatabaseHelper db = new ProjectDatabaseHelper(this);
             List<Project> projects = db.getAllProjects();
-            if(projects.size() > 0){
+            if (projects.size() > 0) {
```

Comment:
In theory, yes. But trusting a member variable for use with a bounds check as opposed to checking against the collection itself sounds like a bad practice.